ethereum/Webaverse.sol

```diff
@@ -117,18 +116,16 @@ contract Webaverse is WebaverseVoucher, OwnableUpgradeable {
                 "Webaverse: Mint transfer failed"
             );
         }
-        _nftContract.mint(to, balance, uri, name, data);
+        _nftContract.mint(to, balance, uri, data);
     }
 
     /**
      * @notice Claims(Mints) the a single NFT with given parameters.
      * @param to The address on which the NFT will be minted(claimed).
-     * @param data The data to store when claim.
      * @param voucher A signed NFTVoucher that describes the NFT to be redeemed.
      **/
     function claim_NFT(
         address to,
-        bytes memory data,
```

Comment:
Don't we need data here?

Reply:
Yes. We don't need data in the claim.
Claiming is the transfer of tokens from the signer to the claimer.